sdks/python/apache_beam/transforms/ptransform_test.py

```diff
@@ -101,6 +101,16 @@ def test_str(self):
         """inputs=('ci',) side_inputs=('cs',)>""",
         str(inputs_tr))
 
+  def test_named_annotations(self):
+    t = beam.Impulse()
+    t.annotations = lambda: {'test': 'value'}
+    named_t = 'Name' >> t
+    self.assertEqual(named_t.annotations(), {'test': 'value'})
+    original_annotations = named_t.annotations()
+    named_t.annotations = lambda: {'another': 'value', **original_annotations}
+    # This is what gets used in apply.
```

Comment:
nit: can weupdate the comment to be a little clear?

Reply:
Done.